src/app/[locale]/projects/detail/[id]/components/LicenseClearing.tsx

```diff
@@ -49,6 +53,47 @@ export default function LicenseClearing({
         router.push(`/projects/generateSourceCode/${projectId}?withSubProjects=${withSubProjects ? "true" : "false"}`)
     }
 
+    const exportProjectSpreadsheet = async ({withLinkedRelease} : {withLinkedRelease : boolean}) => {
```

Comment:
function's parameter should be defined as following
```suggestion
    const exportProjectSpreadsheet = async (withLinkedRelease: boolean) => {
```

Reply:
Addressed !